### DISCUSSION TITLE
It's possible remove color from logs?

### DISCUSSION THREAD

**Discussion by bypper** [crazy-max/diun#1146] [Q&A] [answered]
### Description

I'm sending docker logs to my Rsyslog server, and Diun logs appears with this format by the color:
```
...
#033[90mSun, 07 Apr 2024 18:17:53 CEST#033[0m #033[32mINF#033[0m #033[1mConfiguration loaded from 3 environment variable(s)#033[0m#015
#033[90mSun, 07 Apr 2024 18:17:53 CEST#033[0m #033[32mINF#033[0m #033[1mCron triggered#033[0m#015
...
```
I would like remove colors from Diun logs for clean my logs in Rsyslog.

Best Regards,
Alberto

**Comment by crazy-max** [comment, marked as answer]
You can use `--log-nocolor` or `LOG_NOCOLOR` env var: https://crazymax.dev/diun/usage/command-line/#serve
